extractor/filesystem/filesystem.go

```diff
@@ -237,6 +237,7 @@ func RunFS(ctx context.Context, config *Config, wc *walkContext) ([]*extractor.I
 }
 
 type walkContext struct {
+	//nolint:containedctx
 	ctx               context.Context
```

Comment:
note: it seems like we could probably come close to eliminating this, but that looks like it gets ugly due to needing to match the `fs.WalkDirFunc` type which does not accept a context in its signature